questions.md

```diff
@@ -0,0 +1,2 @@
+# What is PR?
```

Comment:
Pull requests let you tell others about changes you've pushed to a branch in a repository on GitHub. Once a pull request is opened, you can discuss and review the potential changes with collaborators and add follow-up commits before your changes are merged into the base branch. https://help.github.com/en/articles/about-pull-requests